lld/MachO/ObjC.cpp

```diff
@@ -1148,7 +1148,7 @@ void ObjcCategoryMerger::generateCatListForNonErasedCategories(
       assert(nonErasedCatBody && "Failed to relocate non-deleted category");
 
       // Allocate data for the new __objc_catlist slot
-      auto bodyData = newSectionData(target->wordSize);
+      llvm::ArrayRef<uint8_t> bodyData = newSectionData(target->wordSize);
```

Comment:
I think using `auto &` would have also been ok. But since this is a constant, it is preferred to use `ArrayRef<>` over `const SmallVector<> &`.

Reply:
I am not sure I understand, it is using `ArrayRef<>`, or what is the suggested change ?